### PR TITLE
media-gfx/rawtherapee: version bump to 5.5

### PR DESCRIPTION
**Pull request by vivo75**
see announcement, section "News Relevant to Package Maintainers" to the
end of the page:
https://rawtherapee.com/blog/rawtherapee-5.5-released

### CODE REVIEW

**Comment by vivo75** [comment]
see also: https://bugs.gentoo.org/673356

**Comment by vivo75** [comment]
By the way, now that rawtherapee-5.4 is stable 5.2 and 5.3 can be removed.
The force pushed commit add GCO sign-off

**Comment by juippis** [comment]
You have unsorted inherit and *DEPEND lines, but the 5.5 builds and works fine.

**Comment by juippis** [comment]
... why not fix it here also ;)

if you rebase, just rearrange the order of commits so you can squash 3rd into 1st.

File: media-gfx/rawtherapee/rawtherapee-5.5.ebuild [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-inherit cmake-utils toolchain-funcs flag-o-matic
+inherit cmake-utils toolchain-funcs flag-o-matic gnome2-utils xdg-utils
 
+MY_P=${P/_rc/-rc}
 DESCRIPTION="A powerful cross-platform raw image processing program"
 HOMEPAGE="http://www.rawtherapee.com/"
```
Comment (a17r) [inline media-gfx/rawtherapee/rawtherapee-5.5.ebuild]: switch to `https`

File: media-gfx/rawtherapee/rawtherapee-5.5.ebuild [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
-
-MY_P=${P/_rc/-rc}
 SRC_URI="http://rawtherapee.com/shared/source/${MY_P}.tar.xz"
```
Comment (a17r) [inline media-gfx/rawtherapee/rawtherapee-5.5.ebuild]: switch to `https`

**Comment by vivo75** [comment]
sorry guys I give up.
 Time is too dilated for a simple version bump. I'm out, hopefully someone (possibly a dev) will jump in maintaining the ebuild

**Comment by a17r** [comment]
Admittedly I hadn't read the prior discussion, I directly headed into the code. But I don't think we were asking too much.

If your decision is definitive, I can put the package back to unmaintained.

**Pull request closed without merging**